automation/tdx_canonical_setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+CUR_DIR=$(pwd)
+TDX_DIR="$CUR_DIR"/tdx_setup
+source ${CUR_DIR}/tdx_canonical_common
+
+clone_tdx_repo(){
+        echo -e "\n\nCloning TDX git repository with '$BRANCH_NAME' branch.."
+        [ -d $TDX_DIR ] && rm -rf $TDX_DIR
+        git clone -b $BRANCH_NAME https://github.com/canonical/tdx.git $TDX_DIR
+}
+
+setup_tdx_host(){
+        cd $TDX_DIR
+        echo -e "\nSetting up the host for TDX..."
+        sudo ./setup-tdx-host.sh

```

Comment:
Pass TDX_SETUP_ATTESTATION=1 to this script if its a production system

Reply:
this implementation will not work. 
used below command to achieve this
 **sed -i 's/^TDX_SETUP_ATTESTATION=0/TDX_SETUP_ATTESTATION=1/' "$TDX_DIR"/setup-tdx-config**